transports/azure-storage-queues/multi-storageaccount-support.md

```diff
@@ -15,14 +15,13 @@ related:
  - transports/azure-storage-queues/configuration
 ---
 
-Endpoints running on Azure Storage Queues transport using a single storage account is subject to potential throttling once the maximum number of messages is written to the storage account. To overcome this limitation, use multiple storage accounts. To better understand scale out options with storage accounts, it is necessary understand Azure storage account scalability and performance.
-
+Endpoints running on the Azure Storage Queues transport using a single storage account are subject to potential throttling once the maximum number of concurrent requests to the storage account is hit. To overcome this limitation, use multiple storage accounts. To better understand scale out options with storage accounts, it is necessary to understand [Azure storage account scalability and performance](https://docs.microsoft.com/en-us/azure/storage/common/storage-scalability-targets).
```

Comment:
can we tone this down to make it sound less like an obligation but more like an advice? For example

> To overcome this limitation, multiple storage accounts can be used.

Reply:
toned down